src/cascadia/TerminalCore/ICoreSettings.idl

```diff
@@ -34,6 +34,8 @@ namespace Microsoft.Terminal.TerminalControl
         String WordDelimiters;
 
         Boolean ForceVTInput;
+
+        Windows.Foundation.IReference<UInt32> TabColor;

```

Comment:
Shouldn't this be an IControlSetting instead of an ICoreSetting?

Reply:
So, I was imagining this as just another thing that's similar to the title. If VT can modify the "title color" of the terminal, then that's something that's owned by the Core, not the control. It's the same way title "text" works now.